module/VuFind/src/VuFind/Log/LoggerFactory.php

```diff
@@ -475,13 +393,15 @@ public function __invoke(
         // dependencies with other services.
         $callback = function (&$wrapped, $proxy) use ($container, $requestedName) {
             // Now build the actual service:
+            $monologLogger = new MonologLogger('vufind');
             $wrapped = new $requestedName(
-                $container->get(\VuFind\Net\UserIpReader::class)
+                $container->get(UserIpReader::class), // Use imported alias
```

Comment:
I agree that it makes sense to use the imported alias here, but I don't think we need to comment on it:

```suggestion
                $container->get(UserIpReader::class),
```